docs/_docs/installation/kubernetes/generic-configuration.adoc

```diff
@@ -388,6 +388,17 @@ Note that we use the external IP address (LoadBalancer Ingress) of the service.
 include::{javaFile}[tags=connectThinClient, indent=0]
 ----
 
+Thin clients support the partition awareness feature that may speed up operations. Thin clients store a mapping of Ignite nodes to cache partitions and send partition requests directly to the right node.
+A client connected to the Kubernetes cluster through Ingress can't build such a map. Instead, to activate the feature, one should start a client within the cluster and configure it with KubernetesConnectionConfiguration.
```

Comment:
A not for Nikita Safonov, who will be reviewing and editing the doc. 

I would probably put these details on the thin client documentation page. And add a callout to this Kubernetes pages saying that you need to enable a thin-client specific IP finder (providing a reference to the thin client page for mode details).

Reply:
Hi, I'm agree that there should be part of ClientAddressFinder in thin client. As any user can write own implementation of this interface and use it within any (not only k8s env).

@dmagda @Nikita-tech-writer Nikita, if you didn't start work on this ticket yet I will add this to docs. Please let me know you current status on this ticket

Reply:
Hi @timoninmaxim,
Yes, I will start to work on this ticket after the NY holidays, so you can add it to docs for sure. Thank you.

Reply:
@Nikita-tech-writer hi! I've updated docs and push. Please let me know if it is ok or not.

Reply:
Hi @timoninmaxim , I'll give you the feedback shortly.